akka-stream/src/main/resources/reference.conf

```diff
@@ -47,6 +47,9 @@ akka {
     # such as FileSource, FileSink and others.
     file-io-dispatcher = "akka.stream.default-file-io-dispatcher"
 
+    # Blocking IO like OutputStreamSource and InputStreamSink uses by default the same settings
+    blocking-io-dispatcher = "akka.stream.default-file-io-dispatcher"
```

Comment:
Historically only "file io" had a default blocking dispatcher (both in IO and in http (and spray previouwly))